src/utils/rules-proxy.js

```diff
@@ -76,8 +76,8 @@ const createRewriter = async function ({ distDir, projectDir, jwtSecret, jwtRole
 
     if (rules.length !== 0) {
       return (matcher = await redirector.parseJSON(JSON.stringify(rules), {
-        jwtSecret: jwtSecret || 'secret',
-        jwtRole: jwtRole || 'app_metadata.authorization.roles',
```

Comment:
Arguments should be `jwtSecret` and `jwtRoleClaim`.
Default values are already assigned when reading the configuration